app/views/shared/_message_controls.html.haml

```diff
@@ -1,5 +1,5 @@
-= simple_form_for message, remote: !@claim.redeterminable?, authenticity_token: true do |f|
-  = f.input :claim_id, value: @claim.id, as: :hidden
+= form_with model: message, remote: !@claim.redeterminable?, authenticity_token: true do |f|
```

Comment:
@jrmhaig Thanks.

I had already migrated other forms that use the `simple_form` gem to use the `govuk_design_system_formbuilder` gem. Due to the changes in prioritised tickets, I was unable to continue this migration.

Could you migrate the message control form and form elements to use `govuk_design_system_formbuilder`

Reply:
I can have a go. Can you point me towards a similar PR that I can use as a reference? Thanks.

Reply:
See: https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698

It was mostly the Devise related forms that used simple_form.

@jrmhaig